pallets/staking/src/integrations/conviction_voting.rs

```diff
@@ -119,8 +119,14 @@ where
 							position.action_points = position.action_points.saturating_add(points);
 						}
 					}
-					Votes::<T>::mutate(position_id, |voting| {
-						voting.votes.remove(vote_idx);
+					Votes::<T>::mutate_exists(position_id, |voting| {
+						if let Some(v) = voting {
+							v.votes.remove(vote_idx);
+							// Clear the entry from Votes if positionid has no more votes stored
```

Comment:
This is not blocker for me but this comment just says what code does, these kind of comments are not helpful IMO